05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,3 +1,37 @@
+h1 {
+    color: #252b3b;
+    font-size: 1.9em;
+}
+
+/*Overflow: auto helps the page responsive */
+section {
+    display: inline-block;
+    width: 49.75%;
+    overflow: auto;
+
+}
+
```

Comment:
great commenting! you also need to make sure your image isn't expanding past the section here. just add a  img {width:100%;} and that'll do the trick!